src/crimson/os/seastore/async_cleaner.h

```diff
@@ -1290,7 +1408,32 @@ class SegmentCleaner : public SegmentProvider {
       ool_segment_seq_allocator->set_next_segment_seq(seq);
     }
   }
+
+  journal_seq_t get_journal_head() const {
+    if (journal->get_type() ==
+	journal_type_t::SEGMENT_JOURNAL) {
```

Comment:
Same here, why does the meaning of journal_head_committed change?

Reply:
This is to keep track of journal head. I modified this part in the updated commit [2060f8a](https://github.com/ceph/ceph/pull/46725/commits/2060f8a4a932763238b7a7c44ed68746be2dbaa9)

Reply:
See https://github.com/ceph/ceph/pull/46725#discussion_r921847831, I think we don't need to specialize the journal head for RBM.